### DISCUSSION TITLE
请问如何做到表格数据嵌套表格

### DISCUSSION THREAD

**Discussion by PlutoCA** [alibaba/formily#2371] [Q&A 2.x]
请问如何做到ArrayTable嵌套ArrayTable?

![image](https://user-images.githubusercontent.com/31099607/139641259-646f75bf-6314-4913-949b-e6e3a5f45c04.png)
怎样才能把table2的数据放到table1 中.
期望数据结构
 `  table1: {
    a: 1,
    b: 2,
    table2: [{}]
  }`

尝试 ` basePath="table1.table2"`
但是会报错 `Cannot read properties of undefined (reading 'push')`
[demo](https://codesandbox.io/s/great-resonance-gez1v?file=/App.tsx:3153-3178)

感谢~

**Comment by PlutoCA** [comment]
`expandedRowRender: (_: any, index) => (
        <SchemaField schema={addressSchema} name={index} />
      )`

**Reply by PlutoCA** [reply]
这样写 有问题 翻页后index被重置了

**Reply by janryWang** [reply]
翻页问题，discussion里有讨论

**Reply by PlutoCA** [reply]
> 翻页问题，discussion里有讨论

方便贴个地址么， 我没找到😂

**Reply by janryWang** [reply]
我看错问题了，你这个需求用这个库吧 https://github.com/fishedee/antd-formily-boost
